cdc/owner/changefeed_test.go

```diff
@@ -69,7 +70,13 @@ type mockDDLSink struct {
 	// DDLSink
 	ddlExecuting *model.DDLEvent
 	ddlDone      bool
-	mu           struct {
+	// whether to reset ddlDone flag, only for rename table
```

Comment:
how about add a new struct for `rename table` only fields.

Reply:
> how about add a new struct for `rename table` only fields.

Do you mean embedding a struct for `rename table` in `mockDDLSink`, or creating a new DDLSink structure for `rename table`?